packages/components/src/circular-progress-bar/style.scss

```diff
@@ -13,14 +14,18 @@
 	}
 
 	&-empty-circle {
-		stroke: var(--studio-gray-5);
+		stroke: var(--studio-gray-5, $gray-200);
 	}
 
 	&-fill-circle {
-		stroke: var(--color-accent);
+		stroke: var(--color-accent, var(--wp-admin-theme-color));
```

Comment:
What happens in the backport environment where both of these variables are defined. Feels like another good reason to copy-paste IMO 😄

Reply:
Updated in 80ea3c6 to introduce the strokeColor component.